core/ui.libs/src/utils.ts

```diff
@@ -143,10 +147,11 @@ export function cloneDeep(lodash: any, obj: any, ignoreFiles = false) {
  * @param      {string}  contnt   content that should be placed within the file.
  */
 export function downloadObject(fileName: string, content: any): void {
-  const dataStr = 'data:text/json;charset=utf-8,' + encodeURIComponent(JSON.stringify(content));
+  const stringified = JSON.stringify(content, null, 2);
+  const dataStr = `data:text/json;charset=utf-8,${encodeURIComponent(stringified)}`;
```

Comment:
is there any special reason for text/json? I'm pretty sure the standard would be application/json.

```suggestion
  const dataStr = `data:application/json;charset=utf-8,${encodeURIComponent(stringified)}`;
```